bigbluebutton-html5/imports/ui/components/audio/audio-controls/container.jsx

```diff
@@ -38,6 +41,10 @@ const processToggleMuteFromOutside = (e) => {
 };
 
 const handleLeaveAudio = () => {
+  if (getFromUserSettings('bbb_skip_check_audio_on_first_join', APP_CONFIG.skipCheckOnJoin)) {
+    Storage.setItem('getEchoTest', true);
```

Comment:
Storing this in the session means that joining audio after a refresh is no longer considered a first join. In general I support this but it also adds a nuance of difference with #11285 that we might want to address, probably in 11285 itself